xbmc/cores/VideoPlayer/VideoRenderers/RenderManager.h

```diff
@@ -128,7 +129,6 @@ class CRenderManager
   void DeleteRenderer();
   void ManageCaptures();
 
-  void UpdateLatencyTweak();
```

Comment:
leave this method protected, it is not thread safe.
you can call it from CRenderManager::Configure(). no need to pull RM implementation details to VP layer